### PR TITLE
[Fix] fix bug in docs/get_started.md

### PR DESCRIPTION
**Pull request by zytx121**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation
Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification
Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)
Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)
If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by zytx121** [comment]
#10

File: docs/zh_cn/get_started.md [hunk 1]
```diff
@@ -20,7 +20,15 @@ python tools/test.py ${CONFIG_FILE} ${CHECKPOINT_FILE} [optional arguments] --la
 
 Examples:
 
-Inference rotated RetinaNet on DOTA-1.0 dataset. (Please change the [data_root](.../configs/_base_/datasets/dotav1.py) firstly.)
+Inference RotatedRetinaNet on DOTA-1.0 dataset, which can generate compressed files for online [submission](https://captain-whu.github.io/DOTA/). (Please change the [data_root](.../configs/_base_/datasets/dotav1.py) firstly.)
```
Comment (yangxue0827) [inline docs/zh_cn/get_started.md:23]: ../configs/_base_/datasets/dotav1.py

File: docs/zh_cn/get_started.md [hunk 2]
```diff
@@ -20,7 +20,15 @@ python tools/test.py ${CONFIG_FILE} ${CHECKPOINT_FILE} [optional arguments] --la
...
+```shell
+python ./tools/test.py  \
+  configs/rotated_retinanet/rotated_retinanet_obb_r50_fpn_1x_dota_le90.py \
+  checkpoints/SOME_CHECKPOINT.pth --format-only \
+  --eval-options submission_dir=work_dirs/Task1_results
+```
+
+You can change the test set path in the [data_root](.../configs/_base_/datasets/dotav1.py) to the val set or trainval set for the offline evaluation.
```
Comment (yangxue0827) [inline docs/zh_cn/get_started.md:31]: ../configs/_base_/datasets/dotav1.py

**Review by yangxue0827 — APPROVED** [review]
LGTM

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**